config/query/format_boosting.yml

```diff
@@ -0,0 +1,49 @@
+government_index:
```

Comment:
As mentioned above, we'll be removing this list of formats in the B version of the next A/B test, so we'll only have to handle this if we keep the A version.